lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -714,13 +767,12 @@ namespace Js
 #ifdef INTL_ICU
     typedef const char * (*GetAvailableLocaleFunc)(int);
     typedef int (*CountAvailableLocaleFunc)(void);
-    static bool findLocale(const char *langtag, CountAvailableLocaleFunc countAvailable, GetAvailableLocaleFunc getAvailable)
+    static bool findLocale(JavascriptString *langtag, CountAvailableLocaleFunc countAvailable, GetAvailableLocaleFunc getAvailable)
```

Comment:
>JavascriptString *langtag [](start = 27, length = 25)

processing this as JavascriptString instead of const char * is probably some performance overhead. Why switch?

Reply:
This just moves the JavascriptString -> char* conversion from each of the is*LocaleAvailable entrypoints to this method.